code/src/containers/forms/product-form/components/main-info/MainInfo.tsx

```diff
@@ -0,0 +1,90 @@
+import { useState } from "react";
+
+import { productLocales } from "@/containers/forms/product-form/ProductForm.constants";
+import { ProductFormMainInfoSectionProps } from "@/containers/forms/product-form/ProductForm.types";
+
+import AppBox from "@/components/app-box/AppBox";
+import AppButton from "@/components/app-button/AppButton";
+import AppInput from "@/components/app-input/AppInput";
+import AppTypography from "@/components/app-typography/AppTypography";
+
+import cn from "@/utils/cn/cn";
+
+const MainInfo = ({ register, errors }: ProductFormMainInfoSectionProps) => {
+  const [selectedLocal, setSelectedLocal] = useState(0);
+
+  const languageButtons = productLocales.map((item, index) => (
+    <AppButton
+      variant="contained"
+      size="small"
+      key={item.key}
+      className={cn(
+        "product-form__language-button",
+        index === selectedLocal && "product-form__language-button--active",
+        errors.productTranslations?.[index] &&
+          "product-form__language-button--error"
+      )}
+      data-testid="product-form-language-button"
+      onClick={() => setSelectedLocal(index)}
```

Comment:
create a separate handle function for that

Reply:
Thank you, fixed